kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabIp6Ui.java

```diff
@@ -470,6 +470,16 @@ private void refreshForm() {
             refreshFieldsBasedOnInterface(this.selectedNetIfConfig.get());
             refreshFieldsBasedOnSelectedValues();
         }
+
```

Comment:
Doe it make sense to put this into the `refreshFieldsBasedOnSelectedValues` method?

Reply:
Addressed in e3010a7ac9a202fc84f7b99fa5bfc199a567da27